coniql/app.py

```diff
@@ -58,13 +59,21 @@ def main(args=None) -> None:
     parsed_args = parser.parse_args(args)
 
     context = make_context(*parsed_args.config_paths)
-    web.run_app(
-        register_graphql_handlers(
-            app=web.Application(),
-            executor_context=context,
-            executor_http_endpoint="/graphql",
-            subscription_ws_endpoint="/ws",
-            graphiql_enabled=True,
-            engine=make_engine(),
-        )
+    app = register_graphql_handlers(
+        app=web.Application(),
+        executor_context=context,
+        executor_http_endpoint="/graphql",
+        subscription_ws_endpoint="/ws",
+        graphiql_enabled=True,
+        engine=make_engine(),
     )
+    cors = aiohttp_cors.setup(app)
+    for route in app.router.routes():
+        allow_all = {
+            "*": aiohttp_cors.ResourceOptions(
```

Comment:
```suggestion
            # Allow connection from cs-web-proto dev server on same machine
            "localhost:3000": aiohttp_cors.ResourceOptions(
```

Would this work instead? I'd prefer to put in minimal settings rather than open up to everything and forget about it...